test_autolens/point/test_likelihood.py

```diff
@@ -1,23 +1,41 @@
+import pytest
+
 from autolens.point.analysis import AnalysisAllToAllPointSource
 import autolens as al
 import autofit as af
 
 
-def test_likelihood():
+@pytest.fixture
+def grid():
+    return al.Grid2D.uniform(
+        shape_native=(100, 100),
+        pixel_scales=0.05,
+    )
+
+
+@pytest.fixture
+def model():
+    lens = af.Model(
+        al.Galaxy,
+        redshift=0.5,
+        mass=al.mp.Isothermal,
+    )
+    source = af.Model(al.ps.PointSourceChi)
```

Comment:
The `PointSourceChi` object has a special use, which is described as follows:

```
This example above performs point-source fitting and modeling using what is often referred to as an "image-plane 
chi-squared". This means that the multiple image positions predicted by the model are computed in the image-plane
and compared to the observed multiple image positions in the image-plane.

An alternative approach is to perform point-source modeling using a "source-plane chi-squared". This means that the
multiple image positions predicted by the model are computed and compared in the source-plane.

This is often regarded as a less robust way to perform point-source modeling, which has been shown to produce biased
and incorrect lens models. This is because a mass model may predict multiple images that are not observed in the 
image-plane, which the source-plane chi-squared calculation fails to properly account for. 

However, a source-plane chi-squared is much faster to compute than an image-plane chi-squared, as it does not require
triangles to be iteratively traced to the source-plane to find the multiple images. In certain science cases, it is
therefore useful, and therefore supported by **PyAutoLens**. 

The advanced search chaining feature of PyAutoLens allows one initially fit a model quickly using a 
source-plane chi-squared and then switch to an image-plane chi-squared for a robust final lens model.

Using a source-plane chi-squared requires us to simply change the point source model input into the source
galaxy to a `PointSourceChi` object
```

It is basically the object used for when point source tracings are compared in the source-plane and do not need the `TriangleSolver`.

It therefore should not be used for anything which tests image-plane point solving functionality.

The object which does this is just `al.ps.Point`, as the `SourceChi` is a special one-off use-case.